src/content/blam/tags/gbxmodel/readme.md

```diff
@@ -2,7 +2,6 @@
 title: gbxmodel
 template: tag
 stub: true
-tagClass: GBXModel
```

Comment:
This YAML prop is vestigal from before we had the tag list in JS and I was inferring invader structs from page slug.